src/protocol/taiko_alethia/DelayedInclusionStore.sol

```diff
@@ -0,0 +1,100 @@
+//// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+import {IBlobRefRegistry} from "../../blobs/IBlobRefRegistry.sol";
+import {IDelayedInclusionStore} from "../IDelayedInclusionStore.sol";
+
+contract DelayedInclusionStore is IDelayedInclusionStore {
+    struct DueInclusion {
+        bytes32 blobRefHash;
+        uint256 due;
+    }
+
+    /// @notice Emitted when a delayed publication is stored
+    /// @param dueInclusion The delayed publication including
+    /// the blob reference hash and due timestamp
+    /// @param sender The address that stored the delayed publication
+    event DelayedInclusionStored(address indexed sender, DueInclusion dueInclusion);
+
+    /// @notice Emitted when a list of delayed inclusions are processed by the inbox
+    /// @param inclusionsList list of inclusions
+    event DelayedInclusionProcessed(Inclusion[] inclusionsList);
```

Comment:
Perhaps these 2 should be in the interface?

Honestly, I'm not sure we even need an interface (this should be an extension to a base inbox contract or similar), but if we do need it, I'd move the events there.